### ISSUE TITLE
Inconsistent Scrolling behavior for Advanced ROI tab (version 2.5.0-4009)

### ISSUE THREAD

**Issue by sdeshpa** [parallaxinnovations/MicroView#44]
Under the "Advanced ROI" tool, if the "contours" tab is selected, I am unable to scroll up or down with my mouse. However, if the "ROI" tab is selected, I am able to scroll up or down with my mouse.

**Comment by jeremy-gill** [comment]
I'm assuming 'scroll up or down' refers to the Tools&Apps panel on the left hand side of MicroView's main window?  What platform do you see this behaviour on?  It looks like having a scrolled window within a notebook panel doesn't work well when the entire widget is also scrollable - I'll undo the behaviour.

**Comment by sdeshpa** [comment]
Yes, that is correct—the tools and apps panel. I am seeing this on windows
10 x64.
